### PR TITLE
MNT: import handlers from hxntools/bs branch

### PR DESCRIPTION
**Pull request by klauer**
Now using bluesky branch of hxntools, where the handler is separate from the detector class.

New hxntools loads full hdf5 spectra into memory if possible - 200x200 scans now load/save with `fio.make_hdf` in less than 3 minutes.

I've also updated the `li_ophyd_new` environment for the xf03id user to this new version.

### CODE REVIEW

File: pyxrf/model/fileio.py [hunk 1]
```diff
@@ -53,17 +53,14 @@
...
-    #from dataportal import StepScan as ss
-    #from dataportal import DataMuxer as dm
     from databroker.databroker import DataBroker as db, get_table
     # registers a filestore handler for the XSPRESS3 detector
-    from hxntools import detectors as hxndetectors
+    from hxntools import handlers as hxn_handlers
 except ImportError, e:
     db = None
     ss = None
     dm = None
```
Comment (ericdill) [inline pyxrf/model/fileio.py]: while you're in here, want to delete the old references to `ss` and `dm`?

**Comment by ericdill** [comment]
Other than a bordering-on-inconsequential comment, :+1: from me

**Comment by licode** [comment]
Thanks. Looks good to me.

**Comment by klauer** [comment]
@ericdill Removed the old references. Also replaced `dataportal` -> `databroker` in docstrings, hopefully I didn't miss any.

**Comment by ericdill** [comment]
Well now that you're dealing with the dataportal -> databroker renames, can you get the one in https://github.com/NSLS-II-HXN/PyXRF/blob/0539dd40e345f7642a9e0032b49c0be084df8419/pyxrf/view/fileio.enaml?

**Comment by klauer** [comment]
@ericdill Done.

**Comment by ericdill** [comment]
Awesome, thanks

**Comment by licode** [comment]
Great, I will update pyxrf on the beamline.

**Pull request merged**